{"Functions": {"UnrecoverableEntryException()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an UnrecoverableEntryException with no detail message.", "history": "Added in API level 1", "FullName": "public UnrecoverableEntryException ()"}, "UnrecoverableEntryException(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message."]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an UnrecoverableEntryException with the specified detail message, which provides more information about why this exception has been thrown.", "history": "Added in API level 1", "FullName": "public UnrecoverableEntryException (String msg)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_java#security#GeneralSecurityException_E@"], "ClassName": "java.security.UnrecoverableEntryException", "ClassDesc": "This exception is thrown if an entry in the keystore cannot be recovered."}